nf_core/pipeline-template/modules/nf-core/modules/custom/dumpsoftwareversions/main.nf

```diff
@@ -104,3 +95,17 @@ process CUSTOM_DUMPSOFTWAREVERSIONS {
         yaml.dump(module_versions, f, default_flow_style=False)
     """
 }
+
+//
+// Extract name of software tool from process name using $task.process
+//
+def getSoftwareName(task_process) {
+    return task_process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()
+}
+
+//
+// Extract name of module from process name using $task.process
+//
+def getProcessName(task_process) {
+    return task_process.tokenize(':')[-1]
+}
```

Comment:
I keep it so that we don't get an error on these lines:
https://github.com/nf-core/tools/blob/8bf5af620bbbf7d9b01f1b1fd63c0f9bb08df770/nf_core/pipeline-template/modules/nf-core/modules/custom/dumpsoftwareversions/main.nf#L67
https://github.com/nf-core/tools/blob/8bf5af620bbbf7d9b01f1b1fd63c0f9bb08df770/nf_core/pipeline-template/modules/nf-core/modules/custom/dumpsoftwareversions/main.nf#L102
But once you update the version script if we don't need anymore we can just drop it